source/loader/layers/sanitizer/ur_sanitizer_layer.cpp

```diff
@@ -18,8 +18,8 @@ context_t context;
 
 ///////////////////////////////////////////////////////////////////////////////
 context_t::context_t()
-    : interceptor(std::make_unique<SanitizerInterceptor>()),
-      logger(logger::create_logger("sanitizer")) {}
+    : logger(logger::create_logger("sanitizer")),
```

Comment:
We need logger is created before interceptor, otherwise we can not print log info in interceptor constructor.